source/databricks/calculation_engine/tests/calculation/energy/transformations/test_adjust_flex_consumption.py

```diff
@@ -384,3 +384,31 @@ def test_that_the_correct_metering_point_type_is_put_on_the_result(
         ]
         == MeteringPointType.CONSUMPTION.value
     )
+
+
+def test__adjust_flex_consumption__joins_qualities(
```

Comment:
How about?
```suggestion
def test__adjust_flex_consumption__returns_qualities_from_flex_consumption_and_positive_grid_loss(
```

And similar for the other test method.

Reply:
renamed both tests